### ISSUE TITLE
Is it possible to add multiple users into same workspace?

### ISSUE THREAD

**Issue by pgleeson** [OpenSourceBrain/OSBv2#259]
Question from @cengique. 

Currently, no, it's just the person who created the workspace can edit it, and anyone else can open it as readonly. Note: multiple people can have access to the repository on which the workspace is based, so you can update notebooks etc. in your usual git workflow to make them as usable as possible when a new osbv2 workspace is created.

We're working on user/group info now, but fully developed permissions for multiple users to edit workspaces is a few months off. For now it might be best to think of the github repo as the curated/updated/latest version of your model, and any workspace as a place where individual users can view and simulate the model, develop ideas, and maybe share their work if it's interesting.

@filippomc @zsinnema anything else to add?

**Comment by filippomc** [comment]
Yes the idea of collaborators is there from the very first day, but still we need to really focus on the use case that we want to implement and face the challenge of multiple simultaneous edits:
1. Add manually collaborators to a workspace from the existing users
1. Add collaborators by typing the email (many tools send an "invite" email)
1. Clone a workspace (way simpler as we don't need to account for simultaneous file edits). Workspaces may be allowed to be merged back as well.
